tools/autodeployment/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "autodeployment",
+  "version": "0.0.1",
+  "description": "A tool for automatic deployment",
+  "main": "server.js",
+  "dependencies": {
+    "dotenv": "^8.2.0",
+    "github-webhook-handler": "^1.0.0",
+    "shelljs": "^0.8.3"
```

Comment:
You had talked about using pm2.  Are you going to do that in a different PR?

Reply:
Added pm2